xbmc/platform/win32/WIN32Util.cpp

```diff
@@ -1347,53 +1348,13 @@ HDR_STATUS CWIN32Util::ToggleWindowsHDR(DXGI_MODE_DESC& modeDesc)
   return status;
 }
 
-HDR_STATUS CWIN32Util::GetWindowsHDRStatus()
+#ifdef TARGET_WINDOWS_DESKTOP
+static HDR_STATUS GetWindowsHDRStatusWin32()
```

Comment:
For consistency with WinRtUtil maybe include `GetWindowsHDRStatusWin32` method in `CWIN32Util` class and return HDR_STATUS::HDR_UNKNOWN if its called from target store.

I mean:

```c++
HDR_STATUS CWIN32Util::GetWindowsHDRStatusWin32()
{
  HDR_STATUS status = HDR_STATUS::HDR_UNKNOWN;

#ifdef TARGET_WINDOWS_DESKTOP
...
#endif

  return status;
}
```

Reply:
I wondered about the same thing. Done, as a private method, there is no good reason to call it from outside CWIN32Util.